SpreedlyCoreSharp/Request/ProcessPaymentRequest.cs

```diff
@@ -1,53 +1,37 @@
-﻿using System.Xml.Serialization;
+﻿using RestSharp.Serializers;
 using SpreedlyCoreSharp.Domain;
+using System;
 
 namespace SpreedlyCoreSharp.Request
 {
-    [XmlRoot("transaction")]
+    [SerializeAs(Name = "transaction")]
     public class ProcessPaymentRequest
     {
-        [XmlElement("attempt_3dsecure")]
+        [SerializeAs(Name = "attempt_3dsecure")]
         public bool Attempt3DSecure { get; set; }
 
-        [XmlElement("amount")]
+        [SerializeAs(Name = "amount")]
         public decimal Amount { get; set; }
 
-        [XmlIgnore]
```

Comment:
It seems to me that unit conversions should be the responsibility of the user rather than the library. This property would be useful (and missleading) for users working in non-base100 currencies.

If you want to keep it in the library maybe move it into a separate extension.